son/son/__init__.py

```diff
@@ -174,7 +175,7 @@ def trim(content):
                         current_subsection = ''
                         current_subcontent = ''
                     current_subsection = item[1:].strip()
-                elif item.strip() != '':
+                else:
```

Comment:
This is the important change.

When the .md file-parsing code was processing the `#Guidance` sections, it was stripping out empty lines.
This caused `Paragraph 1\n\nParagraph 2` to be converted to `Paragraph 1\nParagraph 2`.

`\n\n` means "new paragraph" in Markdown, but `\n` means nothing at all, so the paragraph gaps were all getting lost!